tests/login.go

```diff
@@ -19,7 +19,9 @@ import (
 // LoginToCirros call LoggedInFedoraExpecter but does not return the expecter
 func LoginToCirros(vmi *v1.VirtualMachineInstance) error {
 	expecter, err := LoggedInCirrosExpecter(vmi)
-	defer expecter.Close()
+	if expecter != nil {
```

Comment:
I agree with @rmohr on this.
We check the return `err` first and only if that passes we perform the defer.
This is a pattern used all over, we should not break it on this case.

Note that if there is no error and we return a nil expecter, that is a bug in the helper function.

Reply:
Thanks for the pointer @EdDev done